src/common/isManifestType.ts

```diff
@@ -0,0 +1,29 @@
+// ==LICENSE-BEGIN==
+// Copyright 2017 European Digital Reading Lab. All rights reserved.
+// Licensed to the Readium Foundation under one or more contributor license agreements.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file exposed on Github (readium) in the project repository.
+// ==LICENSE-END==
+
+import { Metadata as R2Metadata } from "@r2-shared-js/models/metadata";
+import { Publication as R2Publication } from "@r2-shared-js/models/publication";
+
+import { PublicationView } from "./views/publication";
+
+type TMetadataR2Publication = Partial<Pick<R2Metadata, "RDFType">>;
+type TIsRdfType = Partial<Pick<PublicationView, "RDFType">> &
+    Partial<Record<keyof Partial<Pick<R2Publication, "Metadata">>, TMetadataR2Publication>>;
+
+export const isAudiobookFn = (item: TIsRdfType | undefined) =>
+    (item?.RDFType && /http[s]?:\/\/schema\.org\/Audiobook[\/]?$/.test(item?.RDFType)) ||
+    (item?.Metadata?.RDFType && /http[s]?:\/\/schema\.org\/Audiobook[\/]?$/.test(item?.Metadata?.RDFType));
+
+export const isDivinaFn = (item: TIsRdfType | undefined) =>
+    (item?.Metadata?.RDFType &&
+        (/http[s]?:\/\/schema\.org\/ComicStrip[\/]?$/.test(item?.Metadata?.RDFType) ||
+            /http[s]?:\/\/schema\.org\/ComicStory[\/]?$/.test(item?.Metadata?.RDFType) ||
+            /http[s]?:\/\/schema\.org\/VisualNarrative[\/]?$/.test(item?.Metadata?.RDFType))) ||
```

Comment:
What about `VisualArtwork`? (I'm not sure, but this is currently used in `r2-shared-js`, so I need to align the vocabularies)
Note that further down the line, there will be a stricter canonical definition of the "Divina" publication type, which means we will be able to remove the "type sniffing" hacks.

Reply:
Code to align: https://github.com/readium/r2-shared-js/blob/develop/src/parser/divina.ts

Reply:
Discussion context: https://github.com/readium/webpub-manifest/issues/65#issuecomment-672859394

See also: https://github.com/readium/architecture/issues/134